tests_end_to_end/test_dockerserver.py

```diff
@@ -0,0 +1,66 @@
+import requests
+from requests.sessions import HTTPAdapter
+from urllib3 import Retry
+
+from demisto_sdk.commands.common.MDXServer import (DEMISTO_DEPS_DOCKER_NAME,
+                                                   start_docker_MDX_server)
+from demisto_sdk.commands.lint.docker_helper import init_global_docker_client
+
+
+def assert_successful_mdx_call():
```

Comment:
Let's add docstring to all test functions (given when then). It is important to easily understand what we are testing in each test.